modules/msk/mutect1/main.nf

```diff
@@ -0,0 +1,63 @@
+process MUTECT1 {
+    tag "$meta.id"
+    label 'process_low'
+    shell '/bin/sh'
+
+   
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/test:1.1.5':
+        'ghcr.io/msk-access/test:1.1.5' }"
+
+    input:
+    tuple val(case_sample_name), val(control_sample_name), path(case_bam), path(control_bam), path(case_bai), path(control_bai)
```

Comment:
sample name should be pulled out of the meta map like this: https://github.com/mskcc-omics-workflows/modules/blob/develop/modules/msk/polysolver/main.nf#L26.

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc